src/main/webapp/onelive/index.html

```diff
@@ -312,49 +313,9 @@ <h3>Upcoming</h3>
         loadUpcomingEvents(); //load upcoming events
         loadProfiles(); // load profiles
         loadFaqComponent();
+        loadUpcomingEvents(); //load upcoming events
```

Comment:
`loadUpcomingEvents();`

This function has been already called on line 313

Reply:
Fixed